src/EventSubscriber/TaxonEventSubscriber.php

```diff
@@ -25,10 +25,16 @@ public static function getSubscribedEvents()
         return [
             'sylius.taxon.post_create' => 'synchronize',
             'sylius.taxon.post_update' => 'synchronize',
-            'sylius.taxon.post_delete' => 'remove',
```

Comment:
@alexander-schranz all the relevant changes live inside of this and`src/Producer/TaxonMessageProducer.php` file.